### PR TITLE
Add spacing tokens mdx story

### PR DESCRIPTION
**Pull request by narin**
## Description of Change

Adds a Spacing tokens storybook page with a table of tokens and visual example, similar to our color tokens page and [VADS's docs](https://design.va.gov/foundation/spacing-units#primitive).

#### Testing Packages

N/A

### Screenshots/Video

image

## Testing

- [x] Tested on Web

## PR Checklist
Code reviewer validation:
- General
	- [x] PR is linked to ticket(s)
	- [x] PR has `changelog` label applied if it's to be included in the changelog
	- [x] Acceptance criteria: 
		- All satisfied _or_
		- Documented reason for not being performed _or_
		- Split to separate ticket and ticket is linked by relevant AC(s)
	- [x] Above PR sections adequately filled out
	- [x] If any breaking changes, in accordance with the [pre-1.0.0 versioning guidelines](https://github.com/department-of-veterans-affairs/va-mobile-library#versioning-policy): a CU ticket has been created for the VA Mobile App detailing necessary adjustments with the package version that will be published by this ticket
- Code
	- [x] Tests are included if appropriate (or split to separate ticket)
	- [x] New functions have proper TSDoc annotations

## Publish

If changes warrant a new version [per the versioning guidelines](https://github.com/department-of-veterans-affairs/va-mobile-library/blob/main/documentation/versioning.md) and the PR is approved and ready to merge:
- [ ] Merge `main` into branch
- [ ] Merge branch to `main`
- [ ] Verify that [Check Component Integrations](https://github.com/department-of-veterans-affairs/va-mobile-library/actions/workflows/check-component-integrations.yml) workflow ran successfully
- [ ] [Publish new version](https://github.com/department-of-veterans-affairs/va-mobile-library/actions/workflows/publish.yml) after merging

### CODE REVIEW

File: packages/components/storybook/spacing.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Meta } from '@storybook/blocks'
+
+import { SpacingTable } from './SpacingTable'
+
+<Meta title="Design Tokens/Spacing" />
+
```
Comment (TimRoe) [inline packages/components/storybook/spacing.stories.mdx:6]: Might be a special case of getting rid of the whitespace lines here? Looking at the image, it looks like there's actually whitespace above the Spacing heading which we probably don't want

Comment (TimRoe) [reply packages/components/storybook/spacing.stories.mdx:6]: Or maybe we leave it, since the Colors token page also has it.

**Review by TimRoe — APPROVED** [review]
Looks good to me. 

One potential enhancement might be somewhat matching to the Colors page by centering the content and keeping doc alignment at the top in the same width as the token content.

**Review by jessicawoodin — CHANGES_REQUESTED** [review]
Overall, this looks good, but I have some nitpicky feedback:
- It looks like the copy in the screenshot is different than what was provided in the outline. Can you add this copy and the Primitive header above the tokens?
> Spacing tokens are used to maintain a consistent rhythm when applied to margins, padding, and other dimensions.
> 
> [View guidance for spacing tokens on the VA Mobile Documentation Site](https://department-of-veterans-affairs.github.io/va-mobile-app/design/Foundation/Design%20tokens/Spacing)
> 
> **Primitive**

- Can you left align the Token/Value/Examples headers in the table?
- I liked Tim's idea to center the content like the Colors page. Right now, the text gets pretty wide on this Spacing page.

File: packages/components/storybook/spacing.stories.mdx (hunk 1 above)
Comment (narin) [reply packages/components/storybook/spacing.stories.mdx:6]: Test this out and removing the whitespace has no effect on the whitespace above

**Comment by narin** [comment]
- Updated text
- Left aligned table column headings
- Changed table to 100% width
- 
@TimRoe @jessicawoodin Please take a look at the updated screenshot and let me know if there's anything else you'd like to see

**Review by jessicawoodin — APPROVED** [review]
Looks good to me.

**Pull request merged**